posts/do-all-roads-lead-to-rome.md

```diff
@@ -0,0 +1,123 @@
+
+Who benefits?
+
+- **Maintainers/contributors**: Tools like Babel had exposed all their internals to allow the creation of plugins, making it much more difficult to maintain and evolve.
+- **Developers**: For a feature, you only need to touch one configuration, **simplifying the development** without the need of plugins. It also makes it much easier to **reuse settings** between projects.
+
+## Less dependencies
+
+In the JavaScript ecosystem we're used to work with many `devDependencies` for all the tools and their plugins: Jest, Babel, Prettier, Webpack. Besides, each of them have many sub-dependencies.
+
+With Rome, many of these `devDependencies` disappear, moreover [**Rome has zero sub-dependencies**](https://github.com/romefrontend/rome/blob/4fdfc5fb7252085ede73a342d895457328dca46e/package.json#L8).
+
+<figure align="center">
+ <img src="/images/blog-images/dependencies-rome.gif" alt="Webpack vs Rome dependencies tree" class="center" />
+  <figcaption><small>Webpack vs Rome dependencies tree</small></figcaption>
+</figure>
+
+So we have:
+
+- **Stronger security**: The chance to have a vulnerability decreases because we drastically reduce all dependencies.
+- **Better maintenance and evolution of your project**: Before, updating a new release of some tool could broke the integration with another one. Now all is under one well integrated big tool.
+- **Better maintanance and evolution of Rome**: Rome has no dependencies, this means it has a lot of flexibility and a great innovator potential. For example, if they wanted to migrate to another language using wasm, they could do it.
+- **Faster installation**: You won't have to wait so long after doing `npm install`. Less things to install.
+
+## Friendly for beginners
+
+As we've seen in the previous sections, Rome makes it much **easier to start** a project. 
+
+We feel more encouraged to start learning something new if we see that it has an easy start. In the last years starting JavaScript projects without frameworks was really complicated. Simplifying all this tooling will encourage many to learn and enter the world of JavaScript in a much more enjoyable way.
+
+<figure align="center">
+ <img src="/images/blog-images/learning-js.jpg" alt="Picture of a woman learning JavaScript" class="center" />
+  <figcaption><small>Photo by Annie Spratt on Unsplash</small></figcaption>
+</figure>
+
+In order to take the first steps in Rome, you only need to [install it](https://romefrontend.dev/#installation) via `yarn` or `npm`, and then run `rome init` to create the default `.config/rome.rjson`. 
+
+Finally you can use every command of the CLI using `rome [command]`, as `rome check` for linting a set of files, `rome format` to format a single file, `rome start` to start a daemon, etc. You can see all the available commands by running `rome --help`.
```

Comment:
Our formatter is tight to our linter. When you run it, you will see fixes around lint rules and formatting. 

```suggestion
Finally you can use every command of the CLI using `rome [command]`, as `rome check` for linting and formatting a set of files, `rome format` to format a single file, `rome start` to start a daemon, etc. You can see all the available commands by running `rome --help`.
```